modules/ROOT/pages/tgw-manage-arm.adoc

```diff
@@ -0,0 +1,101 @@
+= Manage Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+
+Use Runtime Manager to manage transit gateway attachments.
+
+== Rename a Transit Gateway Attachment
+
+To rename a transit gateway attachment:
```

Comment:
For each of the following subtopics, should we add something about under what circumstances the audience might want or need to do these things: rename, remove, and so on?